### ISSUE TITLE
react-native 0.59 upgrade ISSUE 

### ISSUE THREAD

**Issue by chars1205** [GeekyAnts/vue-native-core#150]
We have upgraded the vue-native project version from 0.55.4 to 0.59.0
However, the following error has occurred
I would appreciate your help.

```
error: bundling failed: Error: Unable to resolve module `./App` from `/Users/chars/workspace/test/test/index.js`: The module `./App` could not be found from `/Users/chars/workspace/test/test/index.js`. Indeed, none of these files exist:
 * `/Users/chars/workspace/test/test/App(.native||.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)`

```

**Comment by neeraj-singh47** [comment]
@chars1205 You need to have metro.config.js file configuration similar to `https://github.com/GeekyAnts/vue-native-cli/blob/master/src/utils/metro.config.js`

**Comment by chars1205** [comment]
@neeraj-singh47 
Where should the configuration of the metro.config.js file be written?
this current react-native info

>  React Native Environment Info:
    System:
      OS: macOS 10.14.4
      CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
      Memory: 277.64 MB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.0 - /usr/local/bin/node
      npm: 6.8.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        API Levels: 23, 26, 27, 28
        Build Tools: 23.0.1, 27.0.3, 28.0.0, 28.0.1, 28.0.2, 28.0.3
        System Images: android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.4 AI-183.5429.30.34.5452501
      Xcode: /undefined - /usr/bin/xcodebuild
    npmPackages:
      react: 16.5.0 => 16.5.0 
      react-native: 0.59.2 => 0.59.2 
    npmGlobalPackages:
      create-react-native-app: 2.0.2
      react-native-cli: 2.0.1

**Comment by neeraj-singh47** [comment]
@chars1205 Are you using the expo version of the vue-native? And can you post the screenshot of the error?

**Comment by chars1205** [comment]
@neeraj-singh47
There was a dependency problem that required using @expo/vector-icons when using native-base.
So we fixed the error using metro-react-native-babel-preset instead of babel-preset-expo.
Thank you for your interest.

**Comment by aljourda** [comment]
I had same problem, I published a template compatible react native 0.59. It seems that you need this file : https://github.com/papsign/vue-native-quickstart/blob/master/metro.config.js in your project root folder. I hope it helps
